core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -16,46 +16,23 @@
  * @fileoverview Controllers for a state's interaction editor.
  */
 
```

Comment:
Please remove the two extra newlines from here, just one newline would be good between the comment and the code.